geniza/corpus/models.py

```diff
@@ -1260,6 +1260,7 @@ def index_data(self):
         counts = defaultdict(int)
         # collect transcription and translation texts for indexing
         transcription_texts = []
+        transcription_texts_regex = []
```

Comment:
Renamed to plaintext—similar to my comment below on `content_text_canvases`, it needs to be a multi valued field, split by canvas and with HTML and newlines removed. The latter could probably be done with solr transformations but I'm not sure it would be possible to shift the canvas splitting logic to solr since canvases are joined pre-solr.